src/ConEmu/VConChild.cpp

```diff
@@ -1685,13 +1685,9 @@ bool CConEmuChild::TrackMouse()
 
 bool CConEmuChild::CheckMouseOverScroll()
 {
-	if (gpSet->isAlwaysShowScrollbar == 0)
+	if (gpSet->isAlwaysShowScrollbar == 0) // 0 -- don't show scrollbar at all
 	{
-		return false; // don't show scrollbar at all
-	}
-	else if (gpSet->isAlwaysShowScrollbar == 1) // 1 -- show always
```

Comment:
Always check if the scrollbar is always displayed. Also, apparently it needs to actually check if the scrollbar is set to auto display.

Reply:
This function/check seems like it may be used for various reasons, I am not 100% certain this doesn't break something else... :(